daemon/rpc_login.go

```diff
@@ -135,14 +141,24 @@ func (r *RPC) loginCommon(customCB customCallbackType) (payload *pb.LoginRespons
 }
 
 // LoginOAuth2 is called when logging in with OAuth2.
-func (r *RPC) LoginOAuth2(in *pb.Empty, srv pb.Daemon_LoginOAuth2Server) error {
+func (r *RPC) LoginOAuth2(in *pb.LoginOAuth2Request, srv pb.Daemon_LoginOAuth2Server) error {
```

Comment:
This is not sending the login success/fail event, is it sent from somewhere else?

Reply:
Yet event duration has to be calculated correctly